### PR TITLE
New reference page: swarm update

### PR DESCRIPTION
**Pull request by marians**
Added reference page for `swarm update`.

Ping @luebken @kordless

### CODE REVIEW

File: content/reference/update.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+date = "2015-03-30"
+type = "page"
+categories = ["Reference", "Swarm CLI Commands"]
+tags = ["swarm update"]
+weight = 70
++++
+
+# Updating a component
+
+The `swarm update` command enables you to update the container version that's running in a specific component. This results in the component being stopped, he new image being pulled from the registry and then the component being restarted.
```
Comment (luebken) [inline content/reference/update.md]: he new image => the new image

File: content/reference/update.md [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+
+```nohighlight
+$ swarm update yourapp/yourservice/yourcomponent
+```
+
+<!-- TODO: Explain what the expected output would be. WOuld the comand wait until the component is restarted successfully?-->
+
+## Updating to a specific image version {#specificversion}
+
+I you want to make sure the component uses a specific version of your image, you would first tag your image using a specific version number and then upload t to the registry. For example:
```
Comment (luebken) [inline content/reference/update.md]: then upload t to the registry => then upload it to the registry

**Comment by luebken** [comment]
LGTM.

**Comment by marians** [comment]
Thanks @luebken  - fixed. Will wait with merge until we can release the CLI.

**Pull request merged**